### PR TITLE
smart-api compliant auto gen openapi schemas

### PR DESCRIPTION
**Pull request by achave11**
The script then translates the swagger document into a openapi format via
a web-api call. Also, it formats the openapi document to be smart-api
compliant, by adding required tags and removing unsupported parameters generated
by the translation.

An updated version of the swagger schemas is also being included, to
reflect the latest changes to de definition files. As well as the translated,
filtered version of the openapi schemas.
#### NOTES: 
* When running the build process, the swagger sources are generated, there seems to be some issues with internal class references in `swagger.yaml`. This could be temporarily approached by running the script as a stand alone process, and thereby, not affecting `$ref` tags in the swagger document. 

- [x] Check that you pass the basic style checks and unit tests by running `mvn clean install`

### CODE REVIEW

File: dockstore-webservice/pom.xml [hunk 1]
```diff
@@ -938,30 +938,39 @@
...
                     <execution>
-                        <id>swagger2openapi)</id>
+                        <id>compile-process-openapi</id>
                         <goals>
                             <goal>exec</goal>
                         </goals>
                         <phase>compile</phase>
                         <configuration>
-                            <executable>swagger2openapi</executable>
+                            <executable>chmod</executable>
```
[the remark this answers is not in the document]
Comment (achave11) [reply dockstore-webservice/pom.xml]: How should we approach the travis build ? Run the executable command in the`before_install` process? Or would it be a problem at all?

[the remark this answers is not in the document]
Comment (denis-yuen) [reply dockstore-webservice/pom.xml]: You should be able to make the file executable, check it into github, and skip this chmod step entirely.

[the remark this answers is not in the document]
Comment (achave11) [reply dockstore-webservice/pom.xml]: Oh, ok. Great. 👍

[the remark this answers is not in the document]
Comment (coverbeck) [reply dockstore-webservice/pom.xml]: The permissions should already be correct, but it still failed on Travis with a permissions error without the chmod. But it works locally. When I pull from Abraham's branch the permissions look correct:

```
rwxr-xr-x  1 charlesoverbeck  staff    1125 Jun 14 10:53 swagger2open.sh
```
I was speculating that maybe Travis has some restrictions. Although your expand_confidential_bundle.sh works.

I'm not sure what's going on.

**Comment by achave11** [comment]
Is this ready?

**Comment by denis-yuen** [comment]
Re-pinged for an additional review

**Comment by achave11** [comment]
Is it okay to merge although `codecov/project` is not passing ?

**Comment by denis-yuen** [comment]
Oh, yeah. Its just too sensitive.

**Pull request merged**